test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -969,6 +969,91 @@ public void VerifyInitScriptsExecution(ProjectTemplate projectTemplate)
             }
         }
 
+        [DataTestMethod]
+        [DynamicData(nameof(GetNetCoreTemplates), DynamicDataSourceType.Method)]
+        [Timeout(DefaultTimeout)]
+        public async Task VerifyCmdFindPackageExactMatchInPMC(ProjectTemplate projectTemplate)
+        {
+            EnsureVisualStudioHost();
+            using (var simpleTestPathContext = new SimpleTestPathContext())
+            {
+                // Arrange
+                var PackageName = "TestPackage";
+                var v100 = "1.0.0";
+                await CommonUtility.CreatePackageInSourceAsync(simpleTestPathContext.PackageSource, PackageName, v100);
+
+                using (var testContext = new ApexTestContext(VisualStudio, projectTemplate, Logger, addNetStandardFeeds: true, simpleTestPathContext: simpleTestPathContext))
+                {
+                    SolutionService solutionService = VisualStudio.Get<SolutionService>();
+                    var nugetConsole = GetConsole(testContext.Project);
+
+                    // Act
+                    nugetConsole.Execute($"find-package {PackageName} -ExactMatch");
+
+                    // Assert
+                    Assert.IsTrue(nugetConsole.IsMessageFoundInPMC(PackageName), $"The package name {PackageName} doesn't show correctly in PMC.");
```

Comment:
You might be better off calling `nugetConsole.GetText()` and then you can assert the result in 1 call.

Reply:
Thanks for your suggestion, I have updated the assert, please help to review them again. Thanks a lot.